themes/SuiteP/css/suitep-base/navbar.scss

```diff
@@ -2345,7 +2344,7 @@
   .globalLinks-desktop .dropdown-toggle {
     margin-left: 0;
     border: none;
-    width: auto;
+    width: 48px;
```

Comment:
Seems like a lot of ```auto``` is replaced by direct ```px``` definitions, is that really required (Less responsive).